scripts/manifest.js

```diff
@@ -44,10 +45,19 @@ const overrides = new Map([
 
 const icons = manifest.map(icon => {
 	const id = typeof icon === 'string' ? icon : icon.symbol;
+	const fileName = `${id.toLowerCase()}.svg`;
+	const svgPath = path.resolve(__dirname, '../svg/color/', fileName);
+	let color;
+	if (fs.existsSync(svgPath)) {
```

Comment:
I don't understand the point of this check. If an icon is in the manifest, it should be guaranteed to exist. If it does not, it should fail loudly, not silently.